src/OrchardCore.Modules/OrchardCore.Media/Views/Shared/MediaFieldEditLocalization.cshtml

```diff
@@ -0,0 +1,12 @@
+@if (!Context.Items.ContainsKey("MediaFieldEditLocalization"))
+{
+    @* Insert only once per request *@
```

Comment:
this was all you needed to do to resolve the issue with multiple ids in the dom.

We use the shapes technique in the other modules, so that the media feature can be disabled, and the shapes will not load a modal for the media app (which is disabled)

What you did here was add a shape wrapper in the media module, which required that the media module was active (so was unecesary)